### PR TITLE
[Cppcheck] Run multiple threads with the `-j` option

### PR DESCRIPTION
**Pull request by suetsuguk**
Related to #498

### CODE REVIEW

File: docs/tools/cplusplus/cppcheck.md [hunk 1]
```diff
@@ -169,3 +171,16 @@ The `config/misra.json` file goes like this:
 ### `bug-hunting`
 
 This option allows you to enable Bug hunting. There is a new "soundy" analysis introduced in Cppcheck-2.0. You can read more about this analysis in the [Cppcheck manual](https://github.com/danmar/cppcheck/blob/main/man/manual.md#bug-hunting).
+
+### `parallel`
+
+This option allows you to run multiple jobs in parallel by using the `-j` option of the `Cppcheck` command. The number of jobs is set to the number of processors on our server where the analyzer is running.
```
Comment (ybiquitous) [inline docs/tools/cplusplus/cppcheck.md:177]: [nitpick] Did you mean `cppcheck` (typo)?

```suggestion
This option allows you to run multiple jobs in parallel by using the `-j` option of the `cppcheck` command. The number of jobs is set to the number of processors on our server where the analyzer is running.
```

File: docs/tools/cplusplus/cppcheck.md [hunk 2]
```diff
@@ -169,3 +171,16 @@ The `config/misra.json` file goes like this:
...
+
+Note that there are some limitations due to the behavior of `Cppcheck`.
```
Comment (ybiquitous) [inline docs/tools/cplusplus/cppcheck.md:179]: [suggestion] I think it needless to use `<code>` here (not the command name).

```suggestion
Note that there are some limitations due to the behavior of Cppcheck.
```

File: docs/tools/cplusplus/cppcheck.md [hunk 3]
```diff
@@ -169,3 +171,16 @@ The `config/misra.json` file goes like this:
...
+
+- The `parallel` option is ignored when the `project` option is specified.
```
Comment (ybiquitous) [inline docs/tools/cplusplus/cppcheck.md:181]: [suggestion] How about making a link as follows?

```suggestion
- The `parallel` option is ignored when the [`project`](#project) option is specified.
```

**Comment by ybiquitous** [comment]
I think this change is almost good. 👍

**Comment by suetsuguk** [comment]
Thank you for pointing out. All three comments are fixed at commit 83a481fa6424b7c2229a50754a1de17d9987c549.

**Review by ybiquitous — COMMENTED** [review]
LGTM 👍

**Review by ybiquitous — APPROVED** [review]

**Pull request merged**